terraform-cloud/exercises/08-variables.tf

```diff
@@ -4,7 +4,8 @@ variable "oauth_connection_name" {
   default = "HashiCat Workshop"
 }
 
-variable "github_organization" {
+// previously github_organizaition but this argument is deprecated
```

Comment:
This is fine however - you add a comment saying what this used to be but it's now this new thing.